app/app.js

```diff
@@ -1,47 +1,37 @@
 window.AstroMessages = []; // For debugging messages
 
+// Astro
+import Application from 'astro/application';
+import AnchoredLayoutPlugin from 'astro/plugins/anchoredLayoutPlugin';
+import NavigationPlugin from 'astro/plugins/navigationPlugin';
+
+import Promise from 'bluebird';
+
 window.run = function() {
-    require([
-        'astro-full',
-        'bluebird',
-        'application',
-        'plugins/webViewPlugin',
-        'plugins/anchoredLayoutPlugin',
-        'plugins/navigationPlugin'
-    ],
-    function(
-        Astro,
-        Promise,
-        Application,
-        WebViewPlugin,
-        AnchoredLayoutPlugin,
-        NavigationPlugin
-    ) {
-
-        // Enter your site url here
-        var baseUrl = 'http://<Your IP Address>:5000';
-
-        // Initialize plugins
-        var mainNavigationPromise = NavigationPlugin.init();
-        var layoutPromise = AnchoredLayoutPlugin.init();
-
-        // Start the app by navigating to the base url
-        mainNavigationPromise.then(function(mainNavigationView){
-            mainNavigationView.navigate(baseUrl);
-        });
-
-        // Use the mainNavigationView as the main content view for our layout
-        Promise.join(layoutPromise, mainNavigationPromise, function(layout, mainNavigationView) {
-            layout.setContentView(mainNavigationView);
-            Application.dismissLaunchImage();
-        });
-
-        // Set our layout as the main view of the app
-        layoutPromise.then(function(layout){
-            Application.setMainViewPlugin(layout);
-        });
-
-    }, undefined, true);
+
+    // Enter your site url here
+    const baseUrl = 'http://<Your IP Address>:5000';
+
+    // Initialize plugins
+    const mainNavigationPromise = NavigationPlugin.init();
+    const layoutPromise = AnchoredLayoutPlugin.init();
```

Comment:
Going to hold off on this since the plan is for this tutorial to disappear... unless the team feels strongly otherwise of course.